osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -232,43 +233,51 @@ private string getBPMRange(Beatmap beatmap)
                 double bpmMax = beatmap.ControlPointInfo.BPMMaximum;
                 double bpmMin = beatmap.ControlPointInfo.BPMMinimum;
 
-                if (Precision.AlmostEquals(bpmMin, bpmMax)) return $"{bpmMin:0}bpm";
+                if (Precision.AlmostEquals(bpmMin, bpmMax)) return $"{bpmMin:0}";
 
-                return $"{bpmMin:0}-{bpmMax:0}bpm (mostly {beatmap.ControlPointInfo.BPMMode:0}bpm)";
+                return $"{bpmMin:0}-{bpmMax:0} (mostly {beatmap.ControlPointInfo.BPMMode:0})";
             }
 
-            public class InfoLabel : Container
+            public class InfoLabel : Container, IHasTooltip
             {
+                public string TooltipText { get; private set; }
+
                 public InfoLabel(BeatmapStatistic statistic)
                 {
+                    TooltipText = statistic.Name;
                     AutoSizeAxes = Axes.Both;
+                    Padding = new MarginPadding { Left = 16 };

```

Comment:
How can I make this better? Without Padding on the left side the Icons get cut in half. This wouldn't be visible but has an effect on when tooltips are displayed when hovering.

Reply:
@Aergwyn I've attached a commit which fixes it. This happened because the icon overflowed the InfoLabel.